src/app/modules/consultations/consultation-profile/consultation-profile.graphql.ts

```diff
@@ -305,6 +335,7 @@ export const SubmitResponseQuery = gql`
               }
               downVoteCount
               responseText
+              responseStatus
```

Comment:
Do we need to disable this on prod?

Reply:
No, this field is present on production API as well.